build-conf/langDefProps01.properties

```diff
@@ -1,4 +1,4 @@
-# Sample of an language definition property files overwrite for group langDefProps01
+# Sample of a language definition properties file override for group langDefProps01
 
 # For details, check out section <Build Property management> in application.properties
```

Comment:
I think, this should point to our doc folder?

Reply:
Do you mean here? https://github.com/IBM/dbb-zappbuild/docs/FilePropertyManagement.md (which doesn't exist yet, but will after the PR is merged)

Reply:
It was not in your update, just saw this in line 3:

> # For details, check out section <Build Property management> in application.properties

I think this does not apply any longer. Maybe we remove it or point to the doc folder.